src/main/java/com/powsybl/openloadflow/sa/AbstractSecurityAnalysis.java

```diff
@@ -290,6 +290,20 @@ protected static void findAllRtcToOperate(List<Action> actions, LfTopoConfig top
         }
     }
 
+    protected static void findAllBranchToOpen(Network network, List<Action> actions, LfTopoConfig topoConfig) {
```

Comment:
If I understand well, should be named findAllBranchToClose?

Reply:
Ah yes, thanks!